ChronoAPI/ChronoLog/include/ChronicleMetadataRPCVisor.h

```diff
@@ -46,49 +48,28 @@ class ChronicleMetadataRPCProxy {
         return g_chronicleMetaDirectory->create_chronicle(name);
     }
```

Comment:
Can you put this extern variable in a header file or make it a member of the class? It a variable that is defined once globally, you shouldn't have to declare it locally in all functions.

Reply:
Do you want to consider reducing the number of LOG statements in the code? A few assertions of invariants required for correct behavior of the function should be enough.

Reply:
> Can you put this extern variable in a header file or make it a member of the class? It a variable that is defined once globally, you shouldn't have to declare it locally in all functions.
I will do that.



> Do you want to consider reducing the number of LOG statements in the code? A few assertions of invariants required for correct behavior of the function should be enough.
These are LOGDs. It should not affect the performance or the readability in releases.